packages/dx-react-grid-bootstrap3/src/plugins/search-panel.jsx

```diff
@@ -0,0 +1,13 @@
+import * as React from 'react';
+import { SearchPanel as SearchPanelBase } from '@devexpress/dx-react-grid';
+import { SearchPanelInput } from '../templates/search-panel-input';
+
+export const SearchPanel = props => (
+  <SearchPanelBase
+    rootComponent={SearchPanelInput}
```

Comment:
As for me, `rootComponent` is some kind of wrapper or panel, not input. Well, I'd rename `rootComponent` to `inputComponent`.

Reply:
One more thought... It can be useful if we introduce two components for search panel. `rootComponent` and `inputComponent`. The first one is just a wrapper(`<div />`). In this case, we can support more [advanced scenarios](https://www.screencast.com/t/SD4of0ib).

Reply:
Having two templates in such simple plugin seem redundant.

Reply:
Ok, I rename this option